### PR TITLE
Make defibrillators a bit easier to use. 

### PR DESCRIPTION
**Pull request by potatoalienof13**
### Purpose

Tells you that the defibrillator needs to be equipped to be used. 

### Changelog:

CL: [Improvement] Adds hint to defib description to make it easier to use.

### CODE REVIEW

**Comment by potatoalienof13** [comment]
think i might have created this pr wrongish.

File: UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab [hunk 1]
```diff
@@ -136,7 +136,7 @@ PrefabInstance:
     - target: {fileID: 9019548994788298775, guid: 89c3f4c525ddf014d8dbe780d74999b1,
         type: 3}
       propertyPath: soundReady.AssetAddress
-      value: Assets/Prefabs/Items/defib/DefibReady.prefab
+      value: null
```
Comment (corp-0) [inline UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: what are these null values?

Comment (potatoalienof13) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: No clue.

Comment (potatoalienof13) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: I think the sounds just might have been removed for some reason?

Comment (potatoalienof13) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: That does not appear to be a real path though.

Comment (corp-0) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: can you check if it works as intended?

Comment (MaxIsJoe) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: If the sounds are null then it won't work as intended and will just throw an error.

Comment (potatoalienof13) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: I am pretty sure that absolutely no audio works now.

Comment (potatoalienof13) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: or maybe i am just doing everything completely wrong

File: UnityProject/Assets/Prefabs/Items/Medical/Devices/Defibrillator.prefab [hunk 2]
```diff
@@ -397,7 +400,7 @@ PrefabInstance:
         type: 3}
       propertyPath: initialDescription
       value: A device that delivers powerful shocks to detachable paddles that resuscitate
-        incapacitated patients.
+        incapacitated patients. Must be worn on the backpack to function.
```
Comment (NoooneyDude) [inline UnityProject/Assets/Prefabs/Items/Medical/Devices/Defibrillator.prefab]:
```suggestion
        incapacitated patients. Must be worn on your back to function.
```

**Review by NoooneyDude — APPROVED** [review]

File: UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab (hunk 1 above)
Comment (corp-0) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: what do you mean "no audio works now" ?

**Review by corp-0 — APPROVED** [review]

File: UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab (hunk 1 above)
Comment (potatoalienof13) [reply UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab]: I think I fucked up addressables by deleting a directory that was not actually something that would be reacquired.

**Pull request merged**